tests/console/check_interactive_flag.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Check that the maintenance updates of the following packages contain a trigger for restarting the update client
+# - libsolv
+# - libzypp
+# - zypper
+# - PackageKit
+# Maintainer: Anna Minou <anna.minou@suse.com>
+# Tags: poo#71443
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Get the maintenance updates of the corresponding packages
+    my $result = script_output q(zypper lp -a | grep -v 'zypper-' | awk '/libsolv/||/libzypp/||/zypper/||/PackageKit/ {print $3}');
```

Comment:
Please don't use awk from perl.

Reply:
Why not? I consider it as a complement to greping the output ^^

Reply:
IMHO adding awk where Perl is perfectly able to do what you want is suboptimal. But since this is just a personal opinion, I'll remove the change request.